tests/resources/redis_databases.json

```diff
@@ -0,0 +1,11 @@
+{
+  "SECRETS": 0,
+  "MODULES": 11,
+  "USERS": 12,
+  "VENDORS": 14,
+  "TEMP_MODULES": 5,
+  "TEST_MODULES": 6,
```

Comment:
Can we get rid of the separate `TEST_MODULES` database now and use db 11 for all the tests (same for vendors)?